commons/zenoh-config/src/lib.rs

```diff
@@ -282,6 +282,7 @@ validated_struct::validator! {
                     client_auth: Option<bool>,
                     client_private_key: Option<String>,
                     client_certificate: Option<String>,
+                    server_name_verification: Option<bool>
```

Comment:
Could you add some documentation on that one in `DEFAULT_CONFIG.json5`, highlighting the dangers associated by not checking that the certificate came from the expected IP/name?